cypress/integration/login.js

```diff
@@ -0,0 +1,36 @@
+import Login from "../../POM/loginPage_PO";
+import { faker } from "@faker-js/faker";
+
+describe("Test for login page", () => {
+  const login = new Login();
+
+  before(function () {
+    cy.visit(Cypress.env("baseUrl"));
```

Comment:
It would be better to insert this code in command.js since this can be used in other test as well